c/tskit/tables.c

```diff
@@ -7730,6 +7765,114 @@ tsk_table_collection_compute_mutation_parents(
     return ret;
 }
 
+int TSK_WARN_UNUSED
+tsk_table_collection_compute_mutation_times(tsk_table_collection_t *self, double *random)
```

Comment:
I think we probably want to have an options parameter here as well - we'll regret it if we don't.

Reply:
https://github.com/tskit-dev/tskit/pull/672/commits/bbf936f6b40e7e38c4e831dddafe59042a34eba5